snowplow-tracker/build.gradle

```diff
@@ -16,15 +16,14 @@ project.ext {
 }
 
 android {
-    compileSdkVersion 31
+    compileSdkVersion 32
     buildToolsVersion "30.0.2"
 
     useLibrary 'android.test.base'
 
     defaultConfig {
-        // With API 24: Remove `androidx.appcompat:appcompat` as it's needed for Consumer<T>.
-        minSdkVersion 21
-        targetSdkVersion 31
+        minSdkVersion 24
+        targetSdkVersion 32
```

Comment:
I am a bit worried about updating this because the last time we did it, some customers weren't able to make use of the tracker because their apps targeted earlier SDK versions. The difference between compile and target SDKs is quite confusing to me but I think we should be more conservative about updating both of them, I would keep both at 31 unless there is a reason we need 32.